graphics/VectorRenderer.h

```diff
@@ -107,6 +109,7 @@ struct DrawStep {
 		shadowFillMode = 0;
 		extraData = 0;
 		scale = 0;
+		shadowIntensity = 0;
```

Comment:
This is obviously the wrong default value. Later in the code you're writing `i += shadowIntensity` which will lead to an endless loop. I think it must be `1`